code/game/objects/effects/decals/misc.dm

```diff
@@ -24,22 +24,38 @@
 	var/range = 3
 	var/hits_left = 3
 	var/range_left = 3
+	var/firstmove = TRUE
```

Comment:
do not collide anything while moving out of parent tile, unless Bump()ing it due to it blocking us